hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/EventTimeMapper.java

```diff
@@ -189,7 +189,7 @@ public EventTimeMapper(EventTimePolicy<? super T> eventTimePolicy) {
      * @return a traverser over the given event and the watermark (if it was due)
      */
     @Nonnull
-    public Traverser<Object> flatMapEvent(@Nonnull T event, int partitionIndex, long nativeEventTime) {
+    public Traverser<Object> flatMapEvent(@Nullable T event, int partitionIndex, long nativeEventTime) {
```

Comment:
It also needs javadoc. E.g.: 
> `@param event the event. If {@code null}, it's equivalent to the behavior of {@link #flatMapIdle()}.

Reply:
it would create a circular definition. as the other `flatMap()` is pointing out back to this method.